Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityProfileCloneWindow.cs

```diff
@@ -404,10 +404,12 @@ private static Object EnsureTargetFolder(Object targetFolder)
             {
                 // AssetDatabase.CreateFolder must be called to create each child of the asset folder
                 // path individually. 
-                // Calling AssetDatabase.CreateFolder("Assets", "MixedRealityToolkit.Generated/CustomProfiles")
-                // generates a folder that looks like "Assets/MixedRealityToolkit.Generated_CustomProfiles".
-                AssetDatabase.CreateFolder("Assets", "MixedRealityToolkit.Generated");
+                
+                // Create the MixedRealityToolkit.Generated folder at the root of the Assets
+                MixedRealityToolkitFiles.TryToCreateGeneratedFolder("Assets/");
```

Comment:
Is the trailing / required here? It's odd to see trailing folder slashes in general but if it's required that's good to know

Reply:
I do not think it is required but I'll test it out to make sure.

Reply:
The trailing / is not required

Reply:
Horray